xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -51,8 +51,8 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The name of this DiagnosticListener.</param>
+        <summary>Creates a new DiagnosticListener. It is a NotificationSource, which means the returned result can be used to  log notifications, but it also has a Subscribe method so notifications can be forwarded arbitrarily. Thus its job is to forward things from the producer to all the listeners (multi-casting). Generally, you should not be making your own DiagnosticListener but use the DiagnosticListener.Default, so that notifications are as 'public' as possible.</summary>
```

Comment:
Should the mentioned types/interfaces be links?

Reply:
```suggestion
        <summary>Creates a new <see cref="T:System.Diagnostics.DiagnosticListener" />. It is a NotificationSource, which means the returned result can be used to  log notifications, but it also has a Subscribe method so notifications can be forwarded arbitrarily. Thus its job is to forward things from the producer to all the listeners (multi-casting). Generally, you should not be making your own <see cref="T:System.Diagnostics.DiagnosticListener" /> but use the DiagnosticListener.Default, so that notifications are as 'public' as possible.</summary>
```